rails_docker/docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
       - '3000:3000'
```

Comment:
let's order the config settings by alphabetical order + here are some improvement taken from Athena:

```
web:
  build: .
  container_name: '#{app_name}_web'
  command: bash -c "bundle check || bundle install && rm -f tmp/pids/server.pid && bundle exec rails s -p 3000 -b '0.0.0.0'"
 depends_on:
    - db
  ports:
    - "3000:3000"
  stdin_open: true
  tty: true
  volumes:
    - '.:/#{app_name}'
    - "ruby-bundle:/bundle"
```